src/testFixtures/workspaceFolder/python3.7-plain-sam-app/hello_world/app.py

```diff
@@ -40,3 +40,21 @@ def lambda_handler(event, context):
             # "location": ip.text.replace("\n", "")
         }),
     }
+
+def lambda_handler_2(event, context):
+    return {
+        "statusCode": 200,
+        "body": json.dumps({
+            "message": "hello from handler 2",
+            # "location": ip.text.replace("\n", "")
+        }),
+    }
+
+def lambda_handler_3(event, context):
+    return {
+        "statusCode": 200,
+        "body": json.dumps({
+            "message": "hello from handler 3",
+            # "location": ip.text.replace("\n", "")
```

Comment:
Why are there lines commented out here? remove them?

Reply:
Why not? The purpose is to have something that might resemble real-world code.

Reply:
That is fine, but can you add a comment that it is intentional? If I saw these tests a year later I would be confused and remove the commented out code probably